### PR TITLE
Add Eastron SDM54 

### PR DESCRIPTION
**Pull request by kiezematze**
adding new meter SDM54; adding entries in README and help for that meter

### CODE REVIEW

File: meters/rs485/sdm54.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+func NewSDM54Producer() Producer {
+	/**
+	 * Opcodes as defined by Eastron SDM54.
+	 * See https://www.eastrongroup.com/eastrongroup/2024/08/21/eastronsdm54seriesusermanualv1.2.pdf
+	 * This is to a large extent a superset of all SDM devices, however there are
+	 * subtle differences (see 220, 230). Some opcodes might not work on some devices.
+	 */
+	ops := Opcodes{
```
Comment (andig) [inline meters/rs485/sdm54.go:20]: Please `gofmt` the code (or use VScode with Go plugin)

Comment (kiezematze) [reply meters/rs485/sdm54.go:20]: Thanks for your comment, I have done that. As this is my first time adapting go code I would need some support: can the for

Comment (kiezematze) [reply meters/rs485/sdm54.go:20]: OK, I figured it out. Is there anything else?

**Comment by andig** [comment]
Great PR, thank you 👍🏻

**Comment by kiezematze** [comment]
thanks a lot, it is my first PR ever on github. MBMD and volkszaehler are great projects.

**Pull request merged**